Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -114,6 +114,32 @@ def check_branch(path, ref)
     opoo "Current branch is #{branch}: do you need to pull inside #{ref}?"
   end
 
+  def formulae_need_bottles?(tap, original_commit)
+    return if Homebrew.args.dry_run?
+
+    if Homebrew::EnvConfig.disable_load_formula?
+      opoo "Can't check if updated bottles are necessary as formula loading is disabled!"
+      return
+    end
+
+    Utils.popen_read("git", "-C", tap.path, "diff-tree",
+                     "-r", "--name-only", "--diff-filter=AM",
+                     original_commit, "HEAD", "--", tap.formula_dir)
+         .lines.each do |line|
+      next unless line.end_with? ".rb\n"
+
+      name = "#{tap.name}/#{File.basename(line.chomp, ".rb")}"
+      begin
+        f = Formula[name]
+      rescue Exception # rubocop:disable Lint/RescueException
```

Comment:
Any chance of making this a bit more specific?

Reply:
This is copied from `brew pull` and I don't really know why it's there. I'll check `git blame` to see if I can find a reason.

Reply:
Oh, no worries, just leave it in that case.